Import guide updates (#48)
neo4j-graph-examples/get-started: documentation/browser-preview.workspace.adoc

= New Browser Preview
// http://localhost:4000/browser-preview.workspace.json

== Introduction

Welcome to the New Browser Preview! 
Browser's querying functionality has been modernized and integrated into Neo4j Aura, and these benefits are now available in the new Browser, bundled with the Neo4j on-prem distribution.

If you use Neo4j Aura you’ll see the same capabilities as this New Browser Preview offered under the **Query** part of the integrated experience. 

> Query and Browser are similar, but Query provides features you might expect from a cloud offering like user content being backed by cloud storage so it’s available wherever you log in. 

Reviewer: ```suggestion
Query and Browser are similar, but Query provides features you can expect from a cloud offering, like user content backed by cloud storage, so it’s available wherever you log in. 
```
